azure-iot-device/azure/iot/device/iothub/abstract_clients.py

```diff
@@ -18,16 +18,16 @@
 class AbstractIoTHubClient(object):
     """A superclass representing a generic client. This class needs to be extended for specific clients."""
 
-    def __init__(self, transport):
+    def __init__(self, pipeline_adapter):
         """Initializer for a generic client.
 
-        :param transport: The transport that the client will use.
+        :param pipeline_adapter: The pipeline_adapter that the client will use.
         """
-        self._transport = transport
+        self._pipeline_adapter = pipeline_adapter
 
     @classmethod
     def from_authentication_provider(cls, authentication_provider, transport_name):
```

Comment:
I guess we better move this over to being `protocol_name` or something

Reply:
I'm not so sure about this.  The word "transport" crosses all SDKs and exists in many of our end-user docs.  I also get the feeling that "protocol" and "transport" are somewhat distinct words and I am loath to use a different word from everyone else.

---
In reply to: [287147297](https://github.com/Azure/azure-iot-sdk-python-preview/pull/95#discussion_r287147297) [](ancestors = 287147297)